include/io/outputs.h

```diff
@@ -5,16 +5,18 @@
 
 // #include "logging.h"
 #include "../common/pfem_extras.hpp"
-#include "gridfunctions.h"
+#include "MFEMContainers.h"
 #include "mesh_extras.hpp"
-#include "named_fields_map.h"
+
+class MFEMProblem;
 
 namespace platypus
 {
 
 class Outputs : public platypus::NamedFieldsMap<mfem::DataCollection>
 {
   friend class ProblemBuilder;
+  friend class ::MFEMProblem;
```

Comment:
Removed (alongside unused GLVis references) - `platypus::Outputs` should probably be removed in a future PR altogether now `MFEMDataCollection` inherits from MOOSE's `FileOutput`, since exports can be managed from there in future.